packages/client/manifest.json

```diff
@@ -4435,6 +4435,48 @@
                 "key": "row"
               }
             ]
+          },
+          {
+            "label": "Side panel fields",
+            "type": "fieldConfiguration",
+            "key": "sidePanelFields",
+            "nested": true,
+            "dependsOn": {
+              "setting": "clickBehaviour",
+              "value": "details"
+            }
+          },
+          {
+            "label": "Show side panel delete button",
```

Comment:
NAB: There are the sort of labels that I think we can shrink - e.g. this one could just be "Show delete button".